doc/2/plugins/guides/events/kuzzle-start/index.md

```diff
@@ -6,6 +6,24 @@ title: kuzzle:start
 
 # kuzzle:start
 
+<DeprecatedBadge version="change-me" />
+
+Triggered when Kuzzle has finished its startup sequence, and is about to accept network connections.
+
+Kuzzle will wait for pipe listeners to return before accepting user requests.
+
+
+## kuzzle:start:before
```

Comment:
Since Kubernetes since to became a real things now, maybe we can copy the probe event names indicating the system state?
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Reply:
That documentation page tells a lot of things, and I'm not really sure about what you're proposing, could you be a bit more specific on that one?

They do speak of `probe readiness" however... "ready" is really the usual way of telling that a system is operational.

Reply:
K8S use 3 types of probes to define application state:
 - startup: the application start booting sequence, useful to kill/restart (or emit an error) if the container is to slow to boot
 - liveness: the application is successfully running (will reboot the container of this probe fail)
 - readyness: the application is ready to receive requests

So we could trigger a first event when Kuzzle starts it's booting sequence, another one when the booting sequence is over and the last one when Kuzzle can accepts requests

(ping @alexandrebouthinon @rolljee )

Reply:
Ok so start, live, ready? 

Why not.